lang/de.json

```diff
@@ -291,6 +295,8 @@
   "SWFFG.EnableForcePool": "Aktiviere Machtwert",
   "SWFFG.EnableForcePoolHint": "Aktiviert das Machtwert-Feld auf dem Charakterblatt",
   "SWFFG.Hyperdrive": "Hyperraumantrieb",
+  "SWFFG.HyperdrivePrimary": "Primary",
```

Comment:
Needs translate tag, here and in es and fr (I'll add in a quick followup commit after merge).